base/version_comparable.go

```diff
@@ -28,18 +26,19 @@ type ComparableVersion struct {
 	epoch, major, minor, patch, other uint8
 	build                             uint16
 	edition                           productEdition
-	strOnce                           sync.Once
 	str                               string
 }
 
-var zeroComparableVersion = &ComparableVersion{
-	epoch:   0,
-	major:   0,
-	minor:   0,
-	patch:   0,
-	other:   0,
-	build:   0,
-	edition: "",
+func zeroComparableVersion() *ComparableVersion {
+	return &ComparableVersion{
+		epoch:   0,
+		major:   0,
+		minor:   0,
+		patch:   0,
+		other:   0,
+		build:   0,
+		edition: "",
+	}
```

Comment:
Should this set `str`, on the off-chance it gets logged somewhere?

Reply:
Actually, this is a case where it would return `0.0.0` so I changed it to maintain compatibility since this can be used in the sharded import feed.